src/controllers/listItemController.js

```diff
@@ -0,0 +1,49 @@
+const dataService = require('../services/dataService/listItemFunctions')
+
+function findListItem(req, res, next) {
+  dataService.findListItem()
+    .then(listsItem => {
+      res.status(200).json(listsItem)
+    })
+    .catch(next)
+}
+
+function createListItem(req, res, next) {
+  dataService.createListItem()
```

Comment:
Not using any of the data that is expected to be provided in the request here. You should be pulling values out of the request body and passing them into `dataService.createListItem()`.